tests/test_vagrant.py

```diff
@@ -641,13 +645,11 @@ def _execute_command_in_vm(v, command):
     Run command via ssh on the test vagrant box.  Returns a tuple of the
     return code and output of the command.
     """
-    vagrant_exe = vagrant.get_vagrant_executable()
-
-    if not vagrant_exe:
+    if not VAGRANT_EXE:
         raise RuntimeError(vagrant.VAGRANT_NOT_FOUND_WARNING)
```

Comment:
Do we still need this test ? after all, if vagrant is not there, earlier tests like test_boxesvm would already have failed

Reply:
true, will fix